pkg/ocm/versions.go

```diff
@@ -77,6 +82,7 @@ func (c *Client) GetVersions(channelGroup string, defaultFirst bool) (versions [
 			Order(order).
 			Page(page).
 			Size(size).
+			Parameter("product", product).
```

Comment:
Should we send the `product` parameter also when it's an empty string?

Reply:
good point, sending now only if it's not empty, it is cleaner